SharedContent/cs-winui-template/WinUI3TemplateCs (Package)/Package.appxmanifest

```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="cfb926cc-c146-49f1-9ac1-cfaf8182bfd7"
+    Publisher="CN=Microsoft Corporation"
+    Version="1.0.0.0" />
+
+  <Properties>
+    <DisplayName>WinUICSTemplate (Package)</DisplayName>
+    <PublisherDisplayName>Microsoft Corporation</PublisherDisplayName>
```

Comment:
I think since these are templates for Microsoft samples, this is ok. The samples are mainly for showcasing how to use specific APIs. I followed the existing UWP sample templates: https://github.com/microsoft/Windows-universal-samples/blob/ca74baf5c9e53955e8d749733b8e3e4839afd472/SharedContent/Templates/UWPSDKSampleCS/Package.appxmanifest#L9